script.js

```diff
@@ -0,0 +1,364 @@
+initializeCalculator();
+function initializeCalculator() {
+  let first = {value: "", unsaved: true, saved: false, integer: true};
+  let second = {value: "", unsaved: true, saved: false, integer: true};
+  let operator = {value: "", unsaved: true, saved: false};
```

Comment:
Why do we have both `saved` and `unsaved` properties? Does one property `saved` not suffice?